docs: remote pinning services (#369)
ipfs/ipfs-docs: docs/how-to/pin-files.md

ipfs repo gc
ipfs cat <foo hash>
```

## Local versus remote pinning

> All the information above assumes that you're pinning items locally — that is, to your local IPFS node. That's the default behavior for IPFS, but it's also possible to pin your files to a _remote pinning service_. These third-party services offer the opportunity for you to pin files not to your own local node, but to nodes that they operate, meaning that you don't need to worry about your own node's available disk space or uptime when it comes to availability.

Reviewer: ```suggestion
All the information above assumes that you're pinning items locally — that is, to your local IPFS node. That's the default behavior for IPFS, but it's also possible to pin your files to a _remote pinning service_. These third-party services give you the opportunity to pin files not to your own local node, but to nodes that they operate. You don't need to worry about your own node's available disk space or uptime.
```

> While you can use a remote pinning service's own GUI, CLI, or other dev tools to manage IPFS files pinned to their service, you can also work directly with pinning services in IPFS itself — meaning that you don't need to learn a pinning service's unique API or other tooling.

Reviewer: ```suggestion
While you can use a remote pinning service's own GUI, CLI, or other dev tools to manage IPFS files pinned to their service, you can also work directly with pinning services using your local IPFS installation — meaning that you don't need to learn a pinning service's unique API or other tooling.
```

ipfs repo gc
ipfs cat <foo hash>
```

> ## Local vs. remote pinning

Reviewer: ```suggestion
## Local versus remote pinning
```

Trying to avoid abbreviations as much as possible, they might cause issues with translation efforts down the road.
